lib/test-utils.js

```diff
@@ -3,12 +3,12 @@
 const fileExists = require('file-exists')
 const resolvePath = require('path').resolve
 const cwd = resolvePath(__dirname, '../')
-const isFile = (filePath) =>
-  fileExists(resolvePath(cwd, `temp/${filePath}`))
+const isFile = (filePath, dir) =>
+  fileExists(resolvePath(cwd, `${dir || 'temp'}/${filePath}`))
```

Comment:
Not sure I feel about adding logic to a template tag like this, but overall I think its fine.